iina/uk.lproj/PrefUIViewController.strings

```diff
@@ -52,14 +52,14 @@
 "EAa-gN-8dL.title" = "Показати залишок часу замість загальної тривалості";
 
 /* Class = "NSTextFieldCell"; title = "When entering Picture-in-Picture:"; ObjectID = "EUd-RD-g0T"; */
-"EUd-RD-g0T.title" = "При переході у режим «Кадр у кадрі»:";
+"EUd-RD-g0T.title" = "При переході у режим «Картинка в картинці»:";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">